TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Domain/Repository/NodeDataRepository.php

```diff
@@ -937,8 +937,15 @@ public function findByProperties($term, $nodeTypeFilter, $workspace, $dimensions
         $queryBuilder = $this->createQueryBuilder($workspaces);
         $this->addDimensionJoinConstraintsToQueryBuilder($queryBuilder, $dimensions);
         $this->addNodeTypeFilterConstraintsToQueryBuilder($queryBuilder, $nodeTypeFilter);
-        // Convert to lowercase, then to json, and then trim quotes from json to have valid JSON escaping.
-        $likeParameter = '%' . trim(json_encode(UnicodeFunctions::strtolower($term), JSON_UNESCAPED_UNICODE), '"') . '%';
+
+        if (is_array($term)) {
```

Comment:
This would only work for a single element array, right? Either we support multiple keys by iterating over each key and add multiple (ANDed) LIKE constraints or we throw an exception here, so the caller know exactly that this is not supported yet (instead of just finding no results or just by chance of ordering in the JSON blob - shudder).